wangle/acceptor/TransportInfo.h

```diff
@@ -24,6 +24,23 @@
 #include <folly/SocketAddress.h>
 #include <folly/portability/Sockets.h>
 
+#ifdef __APPLE__
+#define tcp_info tcp_connection_info
+#define TCP_INFO TCP_CONNECTION_INFO
```

Comment:
Are the rest of these necessary? What in wangle uses them?

Reply:
They're not used by wangle, no. The point of this was to make the same `TCP_*` constants that are available when you include TransportInfo.h on Linux (via the netinet/tcp.h include in folly/portability/Sockets.h) defined on macOS. If this isn't done here, then the calling code that wants to use `tcpinfo` will need to do this on its own, which I guess wouldn't be the end of the world.

I guess the most elegant solution would to have a enum for the TCP states in `TransportInfo` instead of relying on platform-dependent headers.